jobs/nfcens/prep/JEVS_NFCENS_WAVE_GRID2OBS_PREP

```diff
@@ -65,26 +58,23 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 # Define COMIN/COMOUT variables     
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${model_ver})} 
+export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})} 
 export COMINgdas=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}
 
-export COMOUT=${COMOUT:-$(compath.py -o $NET/$evs_ver)}
-export ARCmodel=${COMOUT}/$STEP/$COMPONENT/${RUN}.${INITDATE}/${MODELNAME}/${VERIF_CASE}
+export COMOUT=${COMIN}/${STEP}/${COMPONENT}/${RUN}
+export ARCmodel=${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}
 
 # make the directory where the ARCmodel and gdasnc will be kept
-mkdir -p $ARCmodel
+mkdir -p ${COMOUT}.${INITDATE}/${MODELNAME}/${VERIF_CASE}
 
 export jlogfile=${DATA}/${MODELNAME}_${RUN}_PB2NC.log
 
 ####################################
 # Execute the script                
 ####################################
-env
-$HOMEevs/scripts/${COMPONENT}/${STEP}/exevs_${COMPONENT}_${RUN}_${VERIF_CASE}_${STEP}.sh 1>>$pgmout 2>errfile.${pid}
+$HOMEevs/scripts/nfcens/prep/exevs_nfcens_wave_grid2obs_prep.sh
```

Comment:
@lgannoaa , can you update to use $COMPONENT in place of nfcens, $STEP in place of prep? I would also like to see $RUN in place of wave and VERIF_CASE in place of grid2obs.